src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -437,6 +472,8 @@ def _init_weights(self, module):
         output_hidden_states (`bool`, *optional*):
             Whether or not to return the hidden states of all layers. See `hidden_states` under returned tensors for
             more detail.
+        interpolate_pos_encoding (`bool`, *optional*):
+            Whether to interpolate the pre-trained position encodings.
```

Comment:
I realise this comes from vit - but let's default to `False` here. It's easier than having to deal with null/true/false 

```suggestion
        interpolate_pos_encoding (`bool`, *optional*, `False`):
            Whether to interpolate the pre-trained position encodings.
```